mapbox_tilesets/scripts/cli.py

```diff
@@ -197,6 +197,33 @@ def update(
         raise errors.TilesetsError(r.text)
 
 
+@cli.command("delete")
+@click.argument("tileset", required=True, type=str)
+@click.option("--force", "-f", is_flag=True, help="Circumvents confirmation prompt")
+@click.option("--token", "-t", required=False, type=str, help="Mapbox access token")
+@click.option("--indent", type=int, default=None, help="Indent for JSON output")
+def delete(tileset, token=None, indent=None, force=None):
+    """Delete your tileset.
+
+    tilesets delete <tileset_id>
+    """
+
+    mapbox_api = _get_api()
+    mapbox_token = _get_token(token)
+
+    if not force:
+        click.confirm(
+            "Are you sure you want to delete {0}?".format(tileset), abort=True
+        )
+
+    url = "{0}/tilesets/v1/{1}?access_token={2}".format(
+        mapbox_api, tileset, mapbox_token
+    )
+    r = requests.delete(url)
+    if r.status_code != 200 and r.status_code != 204:
```

Comment:
Could you add some confirmation text here like we have for deleting a tileset source? e.g. 

```py
        click.echo("Tileset deleted.")
```

Reply:
Yep I can do that - one thing we should consider is that the output of most of our commands are all JSON with a few exceptions. Feels odd that _some_ commands just output raw text. 

Something to consider for 2.x though

Reply:
`Tileset deleted.` added in https://github.com/mapbox/tilesets-cli/pull/61/commits/4af26eb544a51a7f5b7df78e07a07e7e35dd7c68!